### PR TITLE
refactor(main): simplify `defaultAction`

### PR DESCRIPTION
**Pull request by Chirab**
- [ ] Add a changelog entry in the section "To Be Released" of CHANGELOG.md

Should we add an entry even if it small refacto ?

The purpose of this PR is only to refacto the DefaultAction function with less lines.
Btw why should this function return an error, as i don't see any a scenario where `Action` of `cli.Command` returns an error for help command.

In the futur, shouldn’t we change the signature of `type ActionFunc func(*Context) error` in `Command` struct since every function just exit and none of them returns an error.

### CODE REVIEW

File: scalingo/main.go [hunk 1]
```diff
@@ -22,25 +22,19 @@ var (
 )
 
 func DefaultAction(c *cli.Context) error {
```
Comment (EtienneM) [inline scalingo/main.go:24]: Can you also make this function private?
```suggestion
func defaultAction(c *cli.Context) error {
```

**Review by EtienneM — COMMENTED** [review]
Thanks for your contribution

> Btw why should this function return an error, as i don't see any a scenario where `Action` of `cli.Command` returns an error for help command.

`app.Action` has the type `cli.ActionFunc` which is `func(*Context) error`.

File: scalingo/main.go [hunk 2]
```diff
@@ -22,25 +22,19 @@ var (
...
-	completeMode := false
-
 	for i := range os.Args {
 		if os.Args[i] == completionFlag {
-			completeMode = true
-			break
-		}
-	}
+			if len(os.Args)-2 > 0 {
```
[the remark this answers is not in the document]
Comment (Chirab) [reply scalingo/main.go]: Thank you for reviewing my pull request. I received your request to add a code comment, but I'm not entirely sure what to write.

I know how important it is to have clear and concise code comments. I was wondering if you have any specific suggestions on what kind of information I should include in the comment. 

Thank you for your help, and I look forward to hearing back from you soon.

**Review by EtienneM — APPROVED** [review]
>  I was wondering if you have any specific suggestions on what kind of information I should include in the comment.

After reading your PR again, I actually think that the code is readable "as is" and do not require an extra comment. Sorry for my previous review and thanks again for your contribution! :clap:

**Pull request merged**